bin/commonmark

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env php
 <?php
 
+fwrite(STDERR, "[DEPRECATED] This command has been deprecated and will be removed in v2.0\n\n");
```

Comment:
Why not raise a silenced deprecation error?

Reply:
I generally prefer that (and should probably add that to the other deprecations here, now that I think of it).  But in this particular case, there's no logger hooked up to this script which will detect them, so I don't think a silenced deprecation error would be noticed.  That being said, I suppose an unsilenced one could work.

Reply:
This has been replaced with a non-silenced deprecation error (and force-pushed).